apk-generator/v2/app/static/js/main.js

```diff
@@ -17,15 +17,24 @@ var $fileProgressHolder = $("#file-progress"),
     $fileProgressVal = $("#file-progress-val");
 
 var $statusMessageHolder = $("#status-message-holder"),
-    $statusMessage = $("#status-message");
+    $statusMessage = $("#status-message"),
+    $buildLog = $("#build-log");
 
 var $errorMessageHolder = $("#error-message-holder"),
     $errorMessage = $("#error-message");
 
 var identifier = null,
     taskId = null,
     pollingWorker = null,
-    downloadUrl = null;
+    downloadUrl = null,
+    socket = null;
+
+socket = io.connect('http://' + document.domain + ':' + location.port);
+
+socket.on('message', function(message) {
+    console.log("[x] Received    : " + message);
```

Comment:
yes thios is just to get update from you. will send everything in place once i get it right.